src/app/traveler/traveler-login/traveler-login.component.html

```diff
@@ -21,7 +21,7 @@
         Username and password not found.
       </div>
       <div class="form-buttons">
```

Comment:
is .form-buttons a class you created? if it is then I would add something like margin: 5px; to the .css to get spacing between the buttons.

Reply:
I think that came from the bootstrap example i used.

Reply:
I fixed the styling on those buttons and added a proper spacing